puppet/common.yaml

```diff
@@ -3,4 +3,21 @@ lookup_options:
     merge: deep
 
 classes:
-  - apt
\ No newline at end of file
+  - nsidc_nfs
+  - nsidc_logrotate
+
+nsidc_nfs::sharemounts:
+  /share/logs/search-solr:
+    project: logs
+    share: "search-solr"
+    options: rw
+
+nsidc_logrotate::rotate:
+  'varlogs':
+    location: /var/log/
+    logs:
+      - 'search-solr-tools.log'
+    postrotate: 'chgrp vagrant /var/log/search-solr-tools.log && chmod g+w /var/log/search-solr-tools.log'
```

Comment:
This line is needed because since vagrant is the user running the script, it needs write access to the log file.  The logrotate plugin does create a new file, but it sets it with `0644` access and uses `root` as the group.